docs/proposals/PROPOSAL_G.md

```diff
@@ -0,0 +1,63 @@
+# Proposal G - Image Compatibility Artifact
+
+The proposal introduces a new `application/vnd.oci.image-compatibility.v1` artifact type.
+Interest groups/communities SHOULD implement tools to address their specific (or common) use cases.
+
+## Artifact Manifest
+
+An artifact manifest MAY contain multiple blobs with different or the same media types, but MUST contain at least one.
+The new artifact type for image compatibility MUST be referenced in OCI documentation.
+
+```json
+{
+  "schemaVersion": 2,
+  "mediaType": "application/vnd.oci.image.manifest.v1+json",
+  "artifactType": "application/vnd.oci.image-compatibility.v1",
+  "config": {
+    "mediaType": "application/vnd.oci.empty.v1+json",
+    "digest": "sha256:44136fa355b3678a1146ad16f7e8649e94fb4fc21fe77e8310c060f61caaff8a",
+    "size": 2
+  },
+  "layers": [
+    {
+      "mediaType": "application/k8s.nfd.image-compatibility.spec.v1alpha1+json",
```

Comment:
Why would nfd (or more generally, the same compatibility group) have multiple?

Reply:
It's not required, but it allows people to bring profiles defined on the artifact side.

Reply:
So the tool to parse this would need to parse the list, find three things, and then download each separately to find a profile? Wouldn't it be easier to just use the structure that I suggested in proposal F and put the profiles in sections, for only one extra API call?

Reply:
To find a profile you would have to download the artifact manifest and lookup for specific blob (could be done with help of annotations - like name). Additionally, that way we also allow people to define more than one compatibility media type if they require.